From 8166a4ede8fadffaf918ed549b58e9719d5a0db9 Mon Sep 17 00:00:00 2001
Message-ID: <8166a4ede8fadffaf918ed549b58e9719d5a0db9.1736059919.git.divya@subvertising.org>
In-Reply-To: <cover.1736059918.git.divya@subvertising.org>
References: <cover.1736059918.git.divya@subvertising.org>
From: Divya Ranjan <divya@subvertising.org>
Date: Sat, 4 Jan 2025 21:17:28 +0000
Subject: [PATCH 24/44] gnu: Add rust-nu-parser-0.88.

* gnu/packages/crates-io.scm (rust-nu-parser-0.88): New variable.
---
 gnu/packages/crates-io.scm | 27 +++++++++++++++++++++++++++
 1 file changed, 27 insertions(+)

diff --git a/gnu/packages/crates-io.scm b/gnu/packages/crates-io.scm
index 5fd3b33b7c..d1050d3860 100644
--- a/gnu/packages/crates-io.scm
+++ b/gnu/packages/crates-io.scm
@@ -47351,6 +47351,33 @@ (define-public rust-nu-json-0.88
     (description "This package provides Fork of serde-hjson.")
     (license license:expat)))
 
+(define-public rust-nu-parser-0.88
+  (package
+    (name "rust-nu-parser")
+    (version "0.88.1")
+    (source
+     (origin
+       (method url-fetch)
+       (uri (crate-uri "nu-parser" version))
+       (file-name (string-append name "-" version ".tar.gz"))
+       (sha256
+        (base32 "07ahgrlqd3bkapxnyiq8mhcpgix50j8qy6wcyyx085d3qzc2gnq1"))))
+    (build-system cargo-build-system)
+    (arguments
+     `(#:cargo-inputs (("rust-bytesize" ,rust-bytesize-1)
+                       ("rust-chrono" ,rust-chrono-0.4)
+                       ("rust-itertools" ,rust-itertools-0.11)
+                       ("rust-log" ,rust-log-0.4)
+                       ("rust-nu-engine" ,rust-nu-engine-0.88)
+                       ("rust-nu-path" ,rust-nu-path-0.88)
+                       ("rust-nu-plugin" ,rust-nu-plugin-0.88)
+                       ("rust-nu-protocol" ,rust-nu-protocol-0.88)
+                       ("rust-serde-json" ,rust-serde-json-1))))
+    (home-page "https://github.com/nushell/nushell/tree/main/crates/nu-parser")
+    (synopsis "Nushell's parser")
+    (description "This package provides Nushell's parser.")
+    (license license:expat)))
+
 (define-public rust-nucleo-0.2
   (package
     (name "rust-nucleo")
-- 
2.47.1

